release/Makefile

```diff
@@ -0,0 +1,105 @@
+# Needed for glob expansion.
```

Comment:
should we not update the docs? have we not a doc for the release process? is not this PR changing it?